tests/tests.rs

```diff
@@ -584,3 +584,25 @@ fn query_or() {
     assert!(results.contains(&(f, "ghi", Or::Right(true))));
     assert!(results.contains(&(g, "jkl", Or::Both(456, false))));
 }
+
+#[test]
+fn column_get() {
+    let mut world = World::new();
+
+    let ent = world.spawn((123, "abc"));
+    let ent2 = world.spawn((true, "hecs"));
+    let column = world.column::<&str>();
+    assert_eq!(*column.get(ent).unwrap(), "abc");
+    assert_eq!(*column.get(ent2).unwrap(), "hecs");
+}
+
+#[test]
+fn column_get_mut() {
+    let mut world = World::new();
+
+    let ent = world.spawn((0, true));
+    let column = world.column_mut::<i32>();
+    let val = column.get(ent).unwrap();
+    *val = 99;
+    assert_eq!(*val, 99);
```

Comment:
We should use `world.get` to verify that the write affected the correct location through a different path.